test/helper.js

```diff
@@ -15,10 +17,17 @@ export async function resetGitRepository() {
 }
 
 async function initRepo() {
-  for (const repoPath of [ VERSIONS_PATH, SNAPSHOTS_PATH ]) {
-    const git = new Git(repoPath);
-    await git.init(); // eslint-disable-line no-await-in-loop
-  }
+  _gitVersion = new Git(VERSIONS_PATH);
+  _gitSnapshot = new Git(SNAPSHOTS_PATH);
+  return Promise.all([ _gitVersion.init(), _gitSnapshot.init() ]);
+}
+
+export function gitVersion() {
+  return _gitVersion;
+}
+
+export function gitSnapshot() {
+  return _gitSnapshot;
```

Comment:
> Why not simply `export gitVersion`? 🤔

Reply:
Because it has to be initialized before, and this is done in a the global `before` of this helper.